### PR TITLE
convert ItUsabilityOperatorHelmChart to JUnit5

### PR DESCRIPTION
**Pull request by xiancao**
Convert ItUsabilityOperatorHelmChart#testDeleteOperatorButNotDomain to JUnit5 test framework.
Move commonly used test methods to CommonUtils.java.
Since I keep getting Jenkins run failure in PR 1647, I will create a new PR and close 1647.
Possible reason for the Jenkins failure is my branch name was too long. 

The Jenkins error in PR 1647:
Running as SYSTEM
[EnvInject] - Loading node environment variables.
Building on master in workspace /var/lib/jenkins/workspace/weblogic-kubernetes-operator-model-in-image-tests
No credentials specified
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/oracle/weblogic-kubernetes-operator # timeout=10
Cleaning workspace
 > git rev-parse --verify HEAD # timeout=10
Resetting working tree
 > git reset --hard # timeout=10
 > git clean -fdx # timeout=10
Fetching upstream changes from https://github.com/oracle/weblogic-kubernetes-operator
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/oracle/weblogic-kubernetes-operator +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse origin/convertItUsabilityOperatorHelm​Chart^{commit} # timeout=10
 > git rev-parse convertItUsabilityOperatorHelm​Chart^{commit} # timeout=10
ERROR: Couldn't find any revision to build. Verify the repository and branch configuration for this job.
Strings match run condition: string 1=[convertItUsabilityOperatorHelm​Chart], string 2=[mii-new-integration-tests]
Run condition [Strings match] preventing perform for step [[Editable Email Notification, Slack Notifications]]
Archiving artifacts
Recording test results
ERROR: Step ‘Publish JUnit test result report’ failed: No test report files were found. Configuration error?
Finished: FAILURE

### CODE REVIEW

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/TestAssertions.java [hunk 1]
```diff
@@ -140,16 +127,23 @@
...
    */
-  public static Callable<Boolean> serviceExists(
-      String serviceName,
-      Map<String, String> label,
-      String namespace
-  ) throws ApiException {
-    return () -> {
-      return Kubernetes.doesServiceExist(serviceName, label, namespace);
-    };
+  public static Callable<Boolean> serviceExists(String serviceName, Map<String, String> label, String namespace) {
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/TestAssertions.java:131]: curious - did you change this to satisfy checkstyle rules?

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java [hunk 2]
```diff
@@ -94,62 +90,44 @@ public static boolean adminNodePortAccessible(String domainUid, String namespace
...
-  public static boolean podStateNotChangedDuringScalingCluster(String podName,
-                                                               String domainUid,
-                                                               String domainNamespace,
-                                                               String podCreationTimestampBeforeScale) {
-
-    // check that the original managed server pod still exists
-    logger.info("Checking that the managed server pod {0} still exists in namespace {1}",
-        podName, domainNamespace);
-    assertTrue(assertDoesNotThrow(() -> doesPodExist(domainNamespace, domainUid, podName),
+  public static boolean podStateNotChanged(String podName,
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java:100]: was anybody else using this?

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java [hunk 3]
```diff
@@ -0,0 +1,655 @@
...
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * The common utility class for tests.
+ */
+public class CommonUtils {
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:88]: maybe call it CommonTestUtils

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java [hunk 4]
```diff
@@ -0,0 +1,655 @@
...
+    // list Helm releases matching operator release name in operator namespace
+    logger.info("Checking operator release {0} status in namespace {1}",
+        OPERATOR_RELEASE_NAME, opNamespace);
+    assertTrue(isHelmReleaseDeployed(OPERATOR_RELEASE_NAME, opNamespace),
+        String.format("Operator release %s is not in deployed status in namespace %s",
+            OPERATOR_RELEASE_NAME, opNamespace));
+    logger.info("Operator release {0} status is deployed in namespace {1}",
+        OPERATOR_RELEASE_NAME, opNamespace);
+
+    // wait for the operator is ready
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:156]: is -> to be

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java [hunk 5]
```diff
@@ -0,0 +1,655 @@
...
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> operatorIsReady(opNamespace),
+            "operatorIsReady failed with ApiException"));
+
+    return opHelmParams;
+  }
+
+  /**
+   * Install NGINX and wait up to 5 minutes until the NGINX pod is ready.
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:172]: 5 -> five

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java [hunk 6]
```diff
@@ -0,0 +1,655 @@
...
+ */
+public class CommonUtils {
+
+  private static ConditionFactory withStandardRetryPolicy =
+      with().pollDelay(2, SECONDS)
+          .and().with().pollInterval(10, SECONDS)
+          .atMost(5, MINUTES).await();
+
+  /**
+   * Install WebLogic operator and wait up to 5 minutes until the operator pod is ready.
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:96]: 5 -> five

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java [hunk 7]
```diff
@@ -0,0 +1,655 @@
...
+                nginxNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> isNginxReady(nginxNamespace), "isNginxReady failed with ApiException"));
+
+    return nginxHelmParams;
+  }
+
+  /**
+   * Create a domain in the specified namespace and wait up to 5 minutes until the domain exists.
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:226]: 5 -> five

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java [hunk 8]
```diff
@@ -0,0 +1,655 @@
...
+   * @param domainUid WebLogic domainUid which is backend to the ingress to be created
+   * @param domainNamespace WebLogic domain namespace in which the domain exists
+   * @param managedServerPort port number of WebLogic domain managed servers
+   * @param clusterNames list of the WebLogic domain cluster names in the domain
+   * @return list of ingress hosts
+   */
+  public static List<String> createIngressForDomainAndVerify(String domainUid,
+                                                             String domainNamespace,
+                                                             int managedServerPort,
+                                                             List<String> clusterNames) {
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:271]: you assume all of the clusters have the same listen port

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java [hunk 9]
```diff
@@ -0,0 +1,655 @@
...
+        .contains(ingressName);
+
+    logger.info("ingress {0} for domain {1} was created in namespace {2}",
+        ingressName, domainUid, domainNamespace);
+
+    return ingressHostList;
+  }
+
+  /**
+   * Check pod is created.
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:294]: was

Comment (markxnelson) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:294]: and you are not really checking it was created, you are checking it exists, which is not exactly the same thing

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java [hunk 10]
```diff
@@ -0,0 +1,655 @@
...
+        .until(assertDoesNotThrow(() -> podExists(podName, domainUid, domainNamespace),
+            String.format("podExists failed with ApiException for pod %s in namespace %s",
+                podName, domainNamespace)));
+  }
+
+  /**
+   * Check pod is ready.
+   *
+   * @param podName pod name to check
+   * @param domainUid the domain in which the pod exists
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:318]: pods do not exist in domains

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java [hunk 11]
```diff
@@ -0,0 +1,655 @@
...
+        .until(assertDoesNotThrow(() -> podReady(podName, domainUid, domainNamespace),
+            String.format("podReady failed with ApiException for pod %s in namespace %s",
+               podName, domainNamespace)));
+  }
+
+  /**
+   * Check service is created.
+   *
+   * @param serviceName service name to check
+   * @param domainNamespace the domain namespace in which the service exists
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:339]: services do not exist in domain namespaces

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java [hunk 12]
```diff
@@ -0,0 +1,655 @@
...
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> serviceExists(serviceName, null, domainNamespace),
+            String.format("serviceExists failed with ApiException for service %s in namespace %s",
+                serviceName, domainNamespace)));
+  }
+
+  /**
+   * Check pod was deleted.
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:356]: you are not checking it was deleted, you are just checking it does not exist

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java [hunk 13]
```diff
@@ -0,0 +1,655 @@
...
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> podDoesNotExist(podName, domainUid, domainNamespace),
+            String.format("podDoesNotExist failed with ApiException for pod %s in namespace %s",
+                podName, domainNamespace)));
+  }
+
+  /**
+   * Check service was deleted.
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:377]: same

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java [hunk 14]
```diff
@@ -0,0 +1,655 @@
...
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> serviceDoesNotExist(serviceName, null, domainNamespace),
+            String.format("serviceDoesNotExist failed with ApiException for service %s in namespace %s",
+                serviceName, domainNamespace)));
+  }
+
+  /**
+   * Create a Docker image for model in image domain.
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:397]: ... for a model ...

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java [hunk 15]
```diff
@@ -0,0 +1,655 @@
...
+  }
+
+  /**
+   * Docker login and push the image to Docker registry.
+   *
+   * @param dockerImage the Docker image to push to registry
+   */
+  public static void dockerLoginAndPushImageToRegistry(String dockerImage) {
+    // docker login, if necessary
+    if (!REPO_USERNAME.equals(REPO_DUMMY_VALUE)) {
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:494]: you really should be checking the registry name matches the dockerImage arg

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java [hunk 16]
```diff
@@ -0,0 +1,655 @@
...
+    boolean secretCreated = assertDoesNotThrow(() -> createSecret(new V1Secret()
+        .metadata(new V1ObjectMeta()
+            .name(secretName)
+            .namespace(namespace))
+        .stringData(secretMap)), "Create secret failed with ApiException");
+    assertTrue(secretCreated, String.format("create secret failed for %s", secretName));
+  }
+
+  /** Scale the WebLogic cluster to specified number of servers.
+   *  And verify the sample app can be accessed through NGINX.
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:531]: don't start a sentence with "And"

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java [hunk 17]
```diff
@@ -0,0 +1,655 @@
...
+   *
+   * @param clusterName the WebLogic cluster name in the domain to be scaled
+   * @param domainUid the domain to which the cluster belongs
+   * @param domainNamespace the domain namespace in which the domain exists
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:535]: the namespace in which the domain exists

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java [hunk 18]
```diff
@@ -0,0 +1,655 @@
...
+      listOfPodCreationTimestamp.add(originalCreationTimestamp);
+    }
+
+    // scale the cluster in the domain
+    logger.info("Scaling cluster {0} of domain {1} in namespace {2} to {3} servers",
+        clusterName, domainUid, domainNamespace, replicasAfterScale);
+    assertThat(assertDoesNotThrow(() -> scaleCluster(domainUid, domainNamespace, clusterName, replicasAfterScale)))
+        .as("Verify scaling cluster {0} of domain {1} in namespace {2} succeeds",
+            clusterName, domainUid, domainNamespace)
+        .withFailMessage("Scaling cluster failed")
```
Comment (markxnelson) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:571]: include more detail, like you do in the as()

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java [hunk 19]
```diff
@@ -0,0 +1,352 @@
...
+
+    // verify the operator pod was deleted
+    logger.info("Checking that operator pod was deleted");
+    checkPodDeleted("weblogic-operator-", null, opNamespace);
+
+    // verify the operator service was deleted
+    logger.info("Checking that operator service was deleted");
+    checkServiceDeleted(OPERATOR_SERVICE_NAME, opNamespace);
+
+    // check that the state of admin server pod in the domain was not changed
```
[the remark this answers is not in the document]
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java:186]: Yes, we should add some wait time here. How long should we wait here? Maybe the operator usual response time for a domain? Any ideas?

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItScaleMiiDomainNginx.java [hunk 20]
```diff
@@ -163,82 +103,116 @@ public static void initAll(@Namespaces(3) List<String> namespaces) {
...
+    // create model in image domain with multiple clusters
+    createMiiDomainWithMultiClusters();
+
+    // create ingress using host based routing
+    List<String> clusterNames = new ArrayList<>();
+    for (int i = 1; i <= NUMBER_OF_CLUSTERS; i++) {
+      clusterNames.add(CLUSTER_NAME_PREFIX + i);
+    }
+    logger.info("Creating ingress for domain {0} in namespace {1}", domainUid, domainNamespace);
+    ingressHostList = createIngressForDomainAndVerify(domainUid, domainNamespace, MANAGED_SERVER_PORT, clusterNames);
```
[the remark this answers is not in the document]
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItScaleMiiDomainNginx.java:127]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItScaleMiiDomainNginx.java [hunk 21]
```diff
@@ -163,82 +103,116 @@ public static void initAll(@Namespaces(3) List<String> namespaces) {
...
+  public void testScaleClusters() {
+
+    for (int i = 1; i <= NUMBER_OF_CLUSTERS; i++) {
 
-    // push image, if necessary
-    if (!REPO_NAME.isEmpty()) {
-      logger.info("docker push image {0} to {1}", miiImage, REPO_NAME);
-      assertTrue(dockerPush(miiImage), String.format("docker push failed for image %s", miiImage));
+      String clusterName = CLUSTER_NAME_PREFIX + i;
+      int numberOfServers = 2 * i - 1;
```
[the remark this answers is not in the document]
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItScaleMiiDomainNginx.java:156]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItScaleMiiDomainNginx.java [hunk 22]
```diff
@@ -814,8 +307,16 @@ private void scaleAndVerifyCluster(String clusterName,
...
+    for (int i = 0; i < ingressHostList.size(); i++) {
+      if (ingressHostList.get(i).contains(clusterName)) {
+        index = i;
+        break;
+      }
+    }
+
     return String.format("curl --silent --show-error --noproxy '*' -H 'host: %s' http://%s:%s/sample-war/index.jsp",
-        domainUid + "." + clusterName + ".test", K8S_NODEPORT_HOST, nodeportshttp);
+        ingressHostList.get(index), K8S_NODEPORT_HOST, nodeportshttp);
```
[the remark this answers is not in the document]
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItScaleMiiDomainNginx.java:319]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java [hunk 23]
```diff
@@ -0,0 +1,352 @@
...
+    createAndVerifyMiiDomain();
+
+    // get the admin server pod original creation timestamp
+    logger.info("Getting admin server pod original creation timestamp");
+    String adminPodOriginalTimestamp =
+        assertDoesNotThrow(() -> getPodCreationTimestamp(domainNamespace, "", adminServerPodName),
+            String.format("getPodCreationTimestamp failed with ApiException for pod %s in namespace %s",
+                adminServerPodName, domainNamespace));
+
+    // get the managed server pod original creation timestamp
```
[the remark this answers is not in the document]
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java:154]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/TestAssertions.java (hunk 1 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/TestAssertions.java:131]: yes

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java (hunk 2 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java:100]: I think so

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java (hunk 3 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:88]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java (hunk 4 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:156]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java (hunk 5 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:172]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java (hunk 6 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:96]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java (hunk 7 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:226]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java (hunk 8 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:271]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java (hunk 9 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:294]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java (hunk 10 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:318]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java (hunk 11 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:339]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java (hunk 12 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:356]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java (hunk 13 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:377]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java (hunk 14 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:397]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java (hunk 15 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:494]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java (hunk 16 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:531]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java (hunk 17 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:535]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java (hunk 18 above)
Comment (xiancao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java:571]: fixed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java (hunk 19 above)
[the remark this answers is not in the document]
Comment (markxnelson) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java:190]: if the operator is gone, then maybe just 10-15 seconds is ok i think -- @rjeberhard ?

**Comment by xiancao** [comment]
Jenkins test result: https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-model-in-image-tests/685/testReport/oracle.weblogic.kubernetes/
https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/97/

**Pull request merged**